api/api_ethereum.go

```diff
@@ -679,3 +728,34 @@ func (api *EthereumAPI) Accounts() []common.Address {
 	// TODO-Klaytn: Not implemented yet.
 	return nil
 }
+
+// rpcMarshalHeader marshal block header as Ethereum compatible format
+func (api *EthereumAPI) rpcMarshalHeader(head *types.Header) map[string]interface{} {
```

Comment:
How about changing the function name so that it knows that it is marshaling to the Ethereum header format?
EthereumAPI is inside the api package, so I think it can prevent confusion in the future if it is differentiated.

Reply:
@jimni1222 
I thought it is clear because the `rpcMarshalHeader` is a method of the struct `EthereumAPI`, so I believe we can guess this method will be do formatting header to Ethereum header.